dagger/build.go

```diff
@@ -41,7 +41,7 @@ func buildAndPushChartToTTL(
 		return "", err
 	}
 
-	valuesYAML = strings.Replace(valuesYAML, "registry: registry.replicated.com", fmt.Sprintf("registry: %s", imageRegistry), 1)
+	valuesYAML = strings.Replace(valuesYAML, "registry: proxy.replicated.com", fmt.Sprintf("registry: %s", imageRegistry), 1)
```

Comment:
This is the one that comes into play during the validate dagger cal (and the validate ci step). `buildAndPushChartToTTL` will get called with a `imageRegistry` val passed to it by the result of `buildAndPushImageToTTL`  which has `ttl.sh` hard coded: https://github.com/replicatedhq/replicated-sdk/blob/b33008f0bae4cce5c8eef660fc553156dc2c8187/dagger/build.go#L28

Reply:
(just random notes for all of us)